src/test/java/BringItOn/test/AbstractTest.java

```diff
@@ -0,0 +1,50 @@
+package BringItOn.test;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.*;
+import BringItOn.page.PastebinMainPage;
+import BringItOn.page.PastebinSubmittedPastePage;
+
+public abstract class AbstractTest {
+    protected WebDriver driver;
+
+    protected static final String NEW_PASTE_TEXT = "git config --global user.name  \"New Sheriff in Town\"\n" +
+            "git reset $(git commit-tree HEAD^{tree} -m \"Legacy code\")\n" +
+            "git push origin master --force";
+    protected static final String NEW_PASTE_TITLE = "how to gain dominance among developers";
+    protected static final String SUBMITTED_PASTE_TITLE_LOCATOR = "//div[@class='info-top']/h1";
+    protected static final String EXPECTED_PASTE_SYNTAX = "Bash";
```

Comment:
Yes, I did exactly this for Cloud Calculator task. Initially, when I was doing Pastebin task, I thought that it would be good to keep the Test class only for the tests themselves, but later realized that the Abstract class isn't intended to hold those variables. Left it like that to emphasize differences between implementations, since I wasn't sure which approach is better.